recipes/napari-nlm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "napari-nlm" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-nlm-{{ version }}.tar.gz
+  sha256: 7ddb0cf593f99b79fdcbe050f4fb4d5a1ade61d1ef4444397c9418415ec2d191
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - gputools
+    - magicgui
+    - napari
+    - numpy
+    - pyopencl ==2022.1.5
```

Comment:
@maweigert, is this pyopencl version the only known one to work, or is there a chance we can relax this a little? Maybe it should only be an upper or lower bound? E.g. `>=2022.1.5` or `<=2022.1.5`?